### DISCUSSION TITLE
feat: organization level dashboard

### DISCUSSION THREAD

**Discussion by nialldevlin1** [langfuse/langfuse#3508] [Ideas]
### Describe the feature or potential improvement

Currently costs can only be tracked within projects. This feature request is to add dashboards at the organization level that present metrics aggregated across all projects (e.g. total cost of inference calls). This would help developers better communicate with stakeholders regarding high level metrics.

**Comment by marcklingen** [comment]
great idea, can you add your upvote and any thoughts here? https://github.com/orgs/langfuse/discussions/1258

closing as duplicate
